packages/@react-aria/selection/src/useTypeSelect.ts

```diff
@@ -41,7 +49,19 @@ export function useTypeSelect(options: TypeSelectOptions): TypeSelectAria {
 
   return {
     typeSelectProps: {
-      onKeyPress: keyboardDelegate.getKeyForSearch ? onKeyPress : null
+      onKeyDown: keyboardDelegate.getKeyForSearch ? onKeyDown : null
     }
   };
 }
+
+function getStringForKey(key: string) {
+  // If the key is of length 1, it is an ASCII value.
+  // Otherwise, if there are no ASCII characters in the key name,
+  // it is a Unicode character.
+  // See https://www.w3.org/TR/uievents-key/
+  if (key.length === 1 || !/^[A-Z]/i.test(key)) {
+    return key;
+  }
+
+  return '';
```

Comment:
will this prevent languages like Chinese?

Reply:
nope, see above comment.